src/main/java/core/basesyntax/service/impl/ReaderServiceCsvImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class ReaderServiceCsvImpl implements ReaderService {

```

Comment:
```suggestion
public class ReaderServiceImpl implements ReaderService {
```
you only read data from file in this service